docusaurus.config.js

```diff
@@ -48,124 +48,126 @@ module.exports = {
             },
             {
               label: 'Stripe',
-              to: '/docs/platform/integration/stripe',
+              to: '/docs/platform/integration/stripe'
             },
             {
               label: 'Mollie',
-              to: '/docs/platform/integration/mollie',
+              to: '/docs/platform/integration/mollie'
             },
             {
               label: 'Wordpress',
-              to: '/docs/platform/integration/wordpress',
-            },
-          ],
+              to: '/docs/platform/integration/wordpress'
+            }
+          ]
         },
         {
           title: 'Community & Help',
           items: [
             {
               label: 'Community Chat',
-              href: 'http://slack.deity.io/',
+              href: 'http://slack.deity.io/'
             },
             {
               label: 'Twitter',
-              href: 'https://twitter.com/deity_commerce',
+              href: 'https://twitter.com/deity_commerce'
             },
             {
               label: 'Instagram',
-              href: 'https://instagram.com/deity_commerce',
+              href: 'https://instagram.com/deity_commerce'
             },
             {
               label: 'YouTube',
-              href: 'https://www.youtube.com/channel/UCCyszDV63yrqFHUY1uWf4mQ',
+              href: 'https://www.youtube.com/channel/UCCyszDV63yrqFHUY1uWf4mQ'
             },
             {
               label: 'Contact Us',
-              href: 'https://deity.io/contact',
-            },
-          ],
+              href: 'https://deity.io/contact'
+            }
+          ]
         },
         {
           title: 'Useful Links',
           items: [
             {
               label: 'Marketing Site',
-              href: 'https://deity.io/',
+              href: 'https://deity.io/'
             },
             {
               label: 'Status',
-              href: 'https://status.deity.io/',
+              href: 'https://status.deity.io/'
             },
             {
               label: 'Blog',
-              href: 'https://medium.com/deity-io',
+              href: 'https://medium.com/deity-io'
             },
             {
               label: 'Privacy Policy',
-              href: 'https://deity.io/privacy-policy',
-            },
-          ],
-        },
+              href: 'https://deity.io/privacy-policy'
+            }
+          ]
+        }
       ],
       logo: {
         alt: 'DEITY Logo',
-        src: 'img/deity-logo.svg',
+        src: 'img/deity-logo.svg'
       },
-      copyright: `Copyright © ${new Date().getFullYear()} DEITY B.V.`,
+      copyright: `Copyright © ${new Date().getFullYear()} DEITY B.V.`
     },
     image: 'img/deity-logo.svg',
 
     algolia: {
+      // https://github.com/facebook/docusaurus/blob/4fad1ce0cd7bf72e07856f72eb5c88e3f153438d/packages/docusaurus-theme-search-algolia/src/validateThemeConfig.ts#L14-L16
```

Comment:
@hobadams I added this comment and a "default appId" below.

Seems like in the prev implementation - they were using some shared appId (but I'd imagine it'll keep working, as they will continue supporting older implementations) 🤷